parlai/core/params.py

```diff
@@ -333,6 +348,15 @@ def parse_args(self, args=None, namespace=None, print_args=True):
         if self.opt.get('datapath'):
             os.environ['PARLAI_DATAPATH'] = self.opt['datapath']
 
+        # nothing changes if model_file does not start with 'models:'
```

Comment:
i think a 1-line comment that says:

'map filenames that start with 'models:' to point to the model zoo'  

would be better?